.github/workflows/build.yml

```diff
@@ -0,0 +1,52 @@
+name: Build the Python app
+run-name: ${{ github.actor }} - ${{ github.ref_name }}
+
+on:
+    push:
+        branches: ["main", "develop"]
+    pull_request:
+        branches: ["main"]
+
+env:
+    PYTHON_VERSION: 3.13
```

Comment:
The specified Python version `3.13` does not exist. The latest stable version is `3.11`. You should update this to a valid version.